apple/RNCWebView.m

```diff
@@ -174,7 +175,7 @@ - (instancetype)initWithFrame:(CGRect)frame
                                                selector:@selector(hideFullScreenVideoStatusBars)
                                                    name:UIWindowDidBecomeHiddenNotification
                                                  object:nil];
-
+      
```

Comment:
Could you clean up all the extra whitespace changes in this PR?

Otherwise this looks good to me. (although I'm not sure why you'd want to turn this feature off ;) )